docs/content/en/docs/reference/artifacts.md

```diff
@@ -28,56 +28,58 @@ However, see [Building node images]({{< relref "#building-node-images">}}) for i
 
 ### Bottlerocket OS images for Bare Metal
 
-Bottlerocket vends its Baremetal variant Images using a secure distribution tool called tuftool. Please refer to [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket image.
+Bottlerocket vends its Baremetal variant Images using a secure distribution tool called `tuftool`. Please refer to [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket image.
 
 ### HookOS (kernel and initial ramdisk) for Bare Metal
 
 kernel:
 ```bash
-https://anywhere-assets.eks.amazonaws.com/releases/bundles/26/artifacts/hook/6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2/vmlinuz-x86_64
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/28/artifacts/hook/6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2/vmlinuz-x86_64
 ```
 
 initial ramdisk:
 ```bash
-https://anywhere-assets.eks.amazonaws.com/releases/bundles/26/artifacts/hook/6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2/initramfs-x86_64
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/28/artifacts/hook/6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2/initramfs-x86_64
 ```
 
 ## vSphere artifacts
 
 ### Bottlerocket OVAs
 
-Bottlerocket vends its VMware variant OVAs using a secure distribution tool called tuftool. Please refer [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket OVA.
+Bottlerocket vends its VMware variant OVAs using a secure distribution tool called `tuftool`. Please refer [Download Bottlerocket node images]({{< relref "#download-bottlerocket-node-images">}}) to download Bottlerocket OVA.
 
 Bottlerocket Tags
 
 OS Family - `os:bottlerocket`
 
 EKS-D Release
 
-1.24 - `eksdRelease:kubernetes-1-24-eks-7`
+1.25 - `eksdRelease:kubernetes-1-25-eks-5`
```

Comment:
Would it be possible to have these names include each link to the actual artifacts?

Reply:
I'm including a command to fetch these artifacts for both VMware and baremetal variants of Bottlerocket. I feel including full URIs will make the docs unnecessarily dense.